Api/Connection.php

```diff
@@ -46,12 +53,12 @@ public function curlCall($url, $headers = [], $body = '', $method = "POST")
             curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
             curl_setopt($curl, CURLOPT_CUSTOMREQUEST, $method);
             curl_setopt($curl, CURLOPT_POSTFIELDS, $body);
-    
+            
```

Comment:
Why are you adding whitespace? :)

Reply:
Some rows had different indentations.

Reply:
So why have you added the blank spaces on row 56?

Reply:
Removed all whitespaces from newline rows.